clients/go/Gopkg.lock

```diff
@@ -1,6 +1,14 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  digest = "1:320e7ead93de9fd2b0e59b50fd92a4d50c1f8ab455d96bc2eb083267453a9709"
+  name = "github.com/asaskevich/govalidator"
```

Comment:
Is this used anywhere? I couldn't find it, and it doesn't seem to be a dependency of the new `ozzo` package; is it a dependency of another package?

Reply:
It's used by ozzo-validation https://github.com/go-ozzo/ozzo-validation/blob/199b55b36bc1af151e6eea487cc92ffa5b460b81/is/rules.go#L12